drizzlepac/hlautils/sourcelist_generation.py

```diff
@@ -648,13 +140,11 @@ def create_sourcelists(obs_info_dict, param_dict):
     Returns
     -------
     """
-    log.info("----------------------------------------------------------------------------------------------------------------------")
+    log.info("-" * 118)
     for key1 in list(obs_info_dict.keys()):
```

Comment:
```suggestion
    for product_type in obs_info_dict:
```
There is no use of the list of keys anywhere else, so that conversion is not needed in the for statement.  Also, calling the variable something more descriptive will make the code easier to follow and to review.  The same suggestion would apply to the next line(for statement) as well.